src/components/AuthorEditCard.vue

```diff
@@ -66,11 +54,7 @@
                     v-on:update:modelValue="
                         $emit('update', 'nameSuffix', $event)
                     "
-                    v-bind:rules="[
-                        (val) =>
-                            (val && val.length > 3) ||
-                            'Please use minimum 3 characters',
-                    ]"
+                    v-bind:rules="[validateNmeSuffix]"
```

Comment:
```suggestion
                    v-bind:rules="[validateNameSuffix]"
```

Reply:
Doh, fixed in https://github.com/citation-file-format/cffinit/pull/187/files#diff-d5f4cc0e7c68bb658cd75092800920fe7f515542639f976ca71a25e6a7b3dee9